src/components/artifacts/MarkdownVisualization.tsx

```diff
@@ -2,5 +2,5 @@ import { Props } from "./Visualization";
 import Markdown from "react-markdown";
 
 export default function MarkdownVisualization({ content }: Props) {
-	return <Markdown className="prose max-w-none">{content}</Markdown>;
+	return <Markdown className="!prose !max-w-none">{content}</Markdown>;
```

Comment:
There was an css override happening, due to an external stylesheet. With the exclamation mark it gets important, so the styles arent overwrittena nymore